docs/cppv2/install/main.md

```diff
@@ -2,17 +2,17 @@
 
 ## Downloading the Library
 
-The preferred method is to "clone" the latest version of the library using the following command
+The preferred method is to "clone" ITensor version 2 using the following commands:
 <div class="commandline"><pre>
 git clone https://github.com/ITensor/ITensor itensor
+cd itensor
+git checkout v2
```

Comment:
If we do the idea I mentioned of changing the default branch, users won't even have to do `git checkout v2`, they'll just have v2 by default. But let's discuss if that's the best approach or not.